olive/engine/footprint.py

```diff
@@ -153,7 +154,7 @@ def mark_pareto_frontier(self):
             self.nodes[k].is_pareto_frontier = cmp_flag
         self.is_marked_pareto_frontier = True
 
-    def get_footprints_by_model_ids(self, model_ids):
+    def create_footprints_by_model_ids(self, model_ids):
```

Comment:
need we rename get_pareto_frontier?

Reply:
Double checked that, after the get_pareto_frontier, olive_engine will change the nodes in some cases. 
Then seems we should rename get_pareto_frontier too.

Updated.